cmd/pageship/app/generate.go

```diff
@@ -0,0 +1,80 @@
+package app
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/oursky/pageship/internal/config"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(generateCmd)
+	generateCmd.AddCommand(generateDockerfileCmd)
+}
+
+var generateCmd = &cobra.Command{
+	Use:   "generate [command]",
+	Short: "Generate files",
+	Args:  cobra.NoArgs, //if unknown command, will return error just like main pageship command
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.Help() //show help if no subcommand supplied
+		return nil
+	},
+}
+
+var Version = "dev"
+var dockerfileTemplate = `FROM ghcr.io/oursky/pageship:v%s
+EXPOSE 8000
+COPY %s /var/pageship
```

Comment:
We also need to copy the config file. A usual config (e.g. bundled SPA) would be something like this:
```
COPY ./pageship.toml /var/pageship/
COPY ./dist /var/pageship/dist/
```